src/components/DayPickerRangeController.jsx

```diff
@@ -254,6 +254,15 @@ export default class DayPickerRangeController extends React.Component {
         );
       }
 
+      if (!startDate && !endDate) {
+        modifiers = this.deleteModifierFromRange(
+          modifiers,
+          this.state.currentMonth,
```

Comment:
I had to change this to:
```
this.state.currentMonth.clone().startOf('month')
``` 

`this.currentMonth` is a specific date and when I ran this locally, it wouldn't clear days in subsequent months properly if this wasn't pegged to the beginning of the `currentMonth`.